### DISCUSSION TITLE
Understand about valkey ?

### DISCUSSION THREAD

**Discussion by velmurugancoder** [valkey-io/valkey#353] [General]
Hi i am new to valkey it is little bit hard to understand the valkey behaviour, only limited documentation is present i want to implement in golang. How to achieve it ?

**Comment by simkusr** [comment]
I guess you can start by using https://github.com/valkey-io/valkey-go and if something occurs it's create issue / PR I think :)

**Comment by zuiderkwast** [comment]
They will be on the website, but before that, the docs can be accessed here: https://github.com/valkey-io/valkey-doc

The manuals and tutorials are under topics/ and commands are under commands/.
